commands/ping.go

```diff
@@ -46,20 +46,20 @@ func runPingCommand(cmd *cobra.Command, args []string) {
 	}
 
 	fmt.Println(color.GreenString("API connection is fine"))
-	fmt.Printf("Ping took %v\n", duration)
+	fmt.Printf("Ping took %d Milliseconds\n", int(duration.Seconds()*1000))
```

Comment:
This could be simplify as 
`fmt.Printf("Ping took %d Milliseconds\n", duration / time.Millisecond)`